crates/subspace-farmer/src/bin/subspace-farmer/main.rs

```diff
@@ -103,6 +103,9 @@ struct DsnArgs {
     /// Defines target total (in and out) connection number that should be maintained.
     #[arg(long, default_value_t = 50)]
     target_connections: u32,
+    /// Known external addresses
+    #[arg(long, alias = "external-address")]
```

Comment:
We didn't use alias for bootstrap nodes, not sure we need it here (also Clap will generate a message suggesting similar CLI argument if you make a typo).

Reply:
I did it because it makes sense to name vector in a plural form and specify an address in a singular form.